backend/services/implementations/testSessionService.ts

```diff
@@ -187,6 +204,78 @@ class TestSessionService implements ITestSessionService {
 
     return testSessionDtos;
   }
+
+  /*
+   * computeTestGrades computes the breakdown and score of a given result and returns the ResultDTO
+   */
+  async computeTestGrades(
+    result: NewResultDTO,
+    testId: string,
+  ): Promise<ResultDTO> {
+    let resultDto: ResultDTO;
+    let questionsCorrect = 0; // the number of questions the student gets correct
+    const studentAnswers: number[] = result.answers; // the array of the student's raw answers
```

Comment:
baaaaa... I think when we defined models, we said that this would always be of type `number`, and if it was short answer it would be the numeric answer and if it was multiple choice, it would be the index. What are thoughts on adding a quick comment here that this is the case? Because I think this logic is not super intuitive.

And also updating the `testSession.model.ts` to say something to that effect as well!